docs/contributing.md

```diff
@@ -91,6 +91,14 @@ New modules should have the following warning at the top of their documentation
 
 We will evaluate incubating modules periodically, and remove the label when appropriate.
 
+### Official modules
+
+In case you are a vendor and are interested in contributing, maintaining and support an Official module to Testcontainers, the same contribution guidelines as outlined above apply.
+As opposed to other Testcontainers modules, Official modules can specify additional [CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) in their module.
+Those code owner act as the vendor-appointed stewards for the module with regards to support and maintenance.
```

Comment:
```suggestion
Those code owners act as the vendor-appointed stewards for the module with regards to support and maintenance.
```

Reply:
Regarding `CODEOWNERS` I'd expect that we'd populate the file with _both_ vendor and core maintainers members/teams. Hopefully this would help enforce the 'both sides must review' rule, even though GitHub's branch protection capabilities aren't quite slick enough to fully enforce it.